core/src/main/java/io/aiven/klaw/helpers/db/rdbms/DeleteDataJdbc.java

```diff
@@ -116,6 +119,27 @@ public String deleteTopicRequest(int topicId, String userName, int tenantId) {
         + " Unable to verify ownership of this request. you may only delete your own requests.";
   }
 
+  public String deleteOperationalRequest(int operationalReqId, String userName, int tenantId) {
+    log.debug("deleteOperationalRequest {}", operationalReqId);
+
+    OperationalRequestID operationalRequestID = new OperationalRequestID();
+    operationalRequestID.setTenantId(tenantId);
+    operationalRequestID.setReqId(operationalReqId);
+
+    Optional<OperationalRequest> operationalRequest =
+        operationalRequestsRepo.findById(operationalRequestID);
+    // UserName is transient and is not set in the database but the requestor is. Both are set to
+    // the userName when the request is created.
+    if (operationalRequest.isPresent()
+        && operationalRequest.get().getRequestor().equals(userName)) {
+      operationalRequest.get().setRequestStatus(RequestStatus.DELETED.value);
+      operationalRequestsRepo.save(operationalRequest.get());
+      return ApiResultStatus.SUCCESS.value;
+    }
+    return ApiResultStatus.FAILURE.value
+        + " Unable to verify ownership of this request. you may only delete your own requests.";
```

Comment:
I think we have this repeated 4 times in this class can you pull out the error message and put it into KlawErrorMessages ?

Reply:
good one. updated